### DISCUSSION TITLE
How to see more detail in the execution of functions

### DISCUSSION THREAD

**Discussion by garuda64** [dbosoft/YaNco#276] [Q&A]
Currently, a function when called is returning an error message indicating an unsuccessful execution.

![imagen](https://github.com/dbosoft/YaNco/assets/17913367/9ee21986-c133-4c85-89b2-31af894f84bf)

How can I validate what the specific error is, since the message that arrives is very generic.

Congratulations again for the excellent work done.

**Comment by garuda64** [comment]
How to check if .SetField("QUANTITY", v.Cantidad.Value) send to Sap the value in format decimal such as: 1.000

**Reply by fw2568** [reply]
If you add a logger like here
https://github.com/dbosoft/YaNco/blob/e1bb32933ae8b56ec3218eb102a26f63aa4d4393/test/SAPSystemTests/Program.cs#L72C1-L74C1

you can see in details of the mapping. Another option is to enable the RFC trace in the connection with the TRACE=3 connection parameter.

**Comment by fw2568** [comment]
For your first question: 
The message you posted is generated on SAP backend. We have no way to check on the client side (YaNco) why SAP is generating this message.

So you need to contact a SAP consultant to find out why the request fails. It is also possible to debug the call within the SAP system.
